From ba0da418774eec685a054e43b71bae5a7a006727 Mon Sep 17 00:00:00 2001
From: Winson Jung <ein_fia@yahoo.com>
Date: Sun, 1 Jun 2014 17:38:49 +0700
Subject: [PATCH] mengubah contoh css dan menambahkan contoh untuk bodynya

---
 src/index.jade | 22 +++++++++++++---------
 1 file changed, 13 insertions(+), 9 deletions(-)

diff --git a/src/index.jade b/src/index.jade
index 76448ad..80e33aa 100644
--- a/src/index.jade
+++ b/src/index.jade
@@ -72,17 +72,21 @@ html
           li Memperkecil ukuran file HTML
        section(data-bespoke-state='emphatic')
        h2 Contoh CSS
-       p #container{
-       p border: 2px dotted #000000;
-          li margin-left:100px;
-          li margin-right:100px;
-          li padding:5px 5px 0px 5px;
-          li }
-       ul
+       p .jdl{
+       p text-shadow:2px 2px 2px #000000;
+       p text-align:center;
+       p font-size:25px;
+       p color:#F60;
+       p font-family:Georgia, "Times New Roman", Times, serif;
+       p }
            
       section(data-bespoke-state='emphatic')
-        h2 Responsive Slides
-        h3 Powered by <a href="https://github.com/markdalgleish/bespoke-scale">bespoke-scale</a>
+       h2 Contoh Body
+       p <div class="jdl">
+       p <p>JenWare... Make It Personal</p>
+       p </div>
+       p <p class="b">Special occasions call for special gifts. The Jenware line of housewares and clothing can be personalized for the ones you love... the happy couple celebrating their new life together... the recent graduate... or a new baby.</p>
+
 
       section(data-bespoke-state='emphatic')
         h2 Bullet List
-- 
1.8.1.msysgit.1

